index.html

```diff
@@ -219,12 +291,108 @@ <h2><i>Kiss And Tell</i></h2>
   				</div>
 				</div> 
              
-            </div><!--/.col-xs-6.col-lg-4-->
+			</div><!--/.col-xs-6.col-lg-4-->
+			
+
           </div><!--/row-->
-			              
+			                <br>
+					<br>
+					<h1 style = "font-size: 45px;text-align:center;" > Movies </h1>
+					
+					
+					
+						<div class="album col-xs-6 col-lg-4">
+						<h3>Neighbours 2</h3>
+						
+						<p><img src="img/neighbors2.jpeg " class="roundmovietiles" ></p>
```

Comment:
Your fork isn't upto date with mine. What you're doing is adding another set of movie tiles to the webpage. 
Update your fork. Then add your `roundtiles` class to that. That will work. (Rn it shows you've added movies on top of movies)